### PR TITLE
fix(minim): build with minim dist browser bundle

### PR DESCRIPTION
**Pull request by opichals**

### CODE REVIEW

File: webpack/base.js [hunk 1]
```diff
@@ -21,6 +21,10 @@ export default {
       '.jsx',
       '.svg',
     ],
+
+    alias: {
+      minim: path.resolve(__dirname, '../node_modules/minim/dist/minim.js'),
```
Comment (kylef) [inline webpack/base.js]: The browser support for minim dist bundle is at https://github.com/refractproject/minim/blob/76949b53b472a1d4048803584b90f35f85ee0546/.babelrc#L5, just wanted to make sure this is similiar to what you support.

Comment (opichals) [reply webpack/base.js]: Thanks for the link! I am actually after "not ie <11".

Would it be a problem to change it?

Comment (kylef) [reply webpack/base.js]: Nope, feel free to test what you need and make a PR. If it's non breaking we can do a patch release.

Comment (opichals) [reply webpack/base.js]: Created https://github.com/refractproject/minim/pull/226 which makes it the same as for phoenix-ui.

**Comment by opichals** [comment]
@kyle updated to use the minim 0.23.3 with IE 11 support.

**Pull request merged**